src/config.ts

```diff
@@ -129,16 +114,17 @@ export const requireGetStore = (opts?: ConfigOpts): StoreInterface => {
  */
 export const requireGetCurrentFromId = (opts?: ConfigOpts): string => {
   const c = getConfig(opts);
-  if (!c.currentFromId) throw MissingUser;
+  if (!c.currentFromId) throw new MissingFromIdConfigError();
   return c.currentFromId;
 };
 
 /**
- * Get the contracts.  Since this is a required field, this is a plain getter.
+ * Get the contracts
```

Comment:
I think there was some confusion about this config option vs the `getContractAddress` method in the contracts module. The Contract config option is actually never required and there are no defaults. It's only intended to be an override for user who would rather use their own solidity contracts instead of ours. Otherwise, we fetch our current contract ABI from the chain, not the config.

At least, that's my understanding. Please, let me know if I have it wrong.